app/src/main/java/com/andrew/tasky/agenda/presentation/screens/event_detail/EventDetailViewModel.kt

```diff
@@ -123,10 +141,12 @@ class EventDetailViewModel @Inject constructor(
         _selectedAttendeeFilterType.value = type
     }
 
-    private val _isAttending = MutableStateFlow(true)
-    val isAttending = _isAttending.asStateFlow()
-    fun switchAttendingStatus() {
-        _isAttending.value = !isAttending.value
+    val isNotAttendeeAndIsEditing = combine(isAttendee, isInEditMode) { isAttendee, isEditing ->
+        !isAttendee && isEditing
+    }
+    val isNotAttendeeAndPhotosIsEmpty = combine(isAttendee, uiEventPhotos) {
+        isAttendee, photos ->
+        !isAttendee && photos.isEmpty()
     }
```

Comment:
I think you'd want to use StateFlows here right? Since you're observing these values in the UI, so you can cache the results